velox/vector/BaseVector.h

```diff
@@ -177,6 +177,20 @@ class BaseVector {
     return type_;
   }
 
+  // Update the type_ with the new type. The new type can have a different
```

Comment:
Use /// for comments on public methods in header files.

"Changes vector type. The new type ..."

> Additionally, note..

"The caller must ensure that this vector is not shared, i.e. singly-referenced."

Reply:
@mbasmanova Updated.